### PR TITLE
Replace usage of deprecated AccumuloInputFormat

### PR DESCRIPTION
**Pull request by lbschanno**
Replace usages of
org.apache.accumulo.core.client.mapreduce.AccumuloInputFormat with the recommended org.apache.accumulo.hadoop.mapreduce.AccumuloInputFormat.

Part of #2443

### CODE REVIEW

File: warehouse/metrics-core/src/main/java/datawave/metrics/analytic/FileByteSummaryLoader.java [hunk 1]
```diff
@@ -105,11 +105,11 @@ public int run(String[] args) throws Exception {
         job.setMapOutputKeyClass(Key.class);
         job.setMapOutputValueClass(Value.class);
         job.setInputFormatClass(AccumuloInputFormat.class);
-        AccumuloInputFormat.setConnectorInfo(job, userName, new PasswordToken(password));
-        AccumuloInputFormat.setInputTableName(job, inputTable);
-        AccumuloInputFormat.setScanAuthorizations(job, Authorizations.EMPTY);
-        AccumuloInputFormat.setZooKeeperInstance(job, ClientConfiguration.loadDefault().withInstance(instance.trim()).withZkHosts(zookeepers.trim()));
-        AccumuloInputFormat.setRanges(job, Collections.singletonList(dayRange));
+
+        Properties clientProperties = Accumulo.newClientProperties().to(instance.trim(), zookeepers.trim()).as(userName, password).build();
```
Comment (apmoriarty) [inline warehouse/metrics-core/src/main/java/datawave/metrics/analytic/FileByteSummaryLoader.java]: Can we disable formatting and put builder options on separate lines, like so?
```java
// @formatter:off
AccumuloInputFormat.configure()
    .clientProperties(clientProperties)
    .table(inputTable)
    .auths(Authorizations.EMPTY)
    .ranges(Collections.singletonList(dayRange))
    .store(job);
// @formatter:on
```

Comment (apmoriarty) [reply warehouse/metrics-core/src/main/java/datawave/metrics/analytic/FileByteSummaryLoader.java]: Comment applies throughout.

**Review by apmoriarty — DISMISSED** [review]

File: warehouse/metrics-core/src/main/java/datawave/metrics/analytic/FileByteSummaryLoader.java (hunk 1 above)
Comment (lbschanno) [reply warehouse/metrics-core/src/main/java/datawave/metrics/analytic/FileByteSummaryLoader.java]: No problem, done.

**Review by apmoriarty — APPROVED** [review]

File: warehouse/metrics-core/src/main/java/datawave/metrics/analytic/FileByteSummaryLoader.java (hunk 1 above)
Comment (apmoriarty) [reply warehouse/metrics-core/src/main/java/datawave/metrics/analytic/FileByteSummaryLoader.java]: Thank you so much!

Comment (lbschanno) [reply warehouse/metrics-core/src/main/java/datawave/metrics/analytic/FileByteSummaryLoader.java]: No problem!

**Review by avgAGB — APPROVED** [review]

**Review by alerman — APPROVED** [review]

**Review by hlgp — APPROVED** [review]

**Review by ivakegg — APPROVED** [review]

**Review by keith-turner — APPROVED** [review]
If these changes are merged it will help minimize diffs with the accumulo4 branch.  I made similar changes in the accumulo4 branch because I did not realize these changes existed.  Would be nice to get these changes into integration and then merged into the accumulo4 branch.

**Pull request merged**